pkg/common/bundleutil/bundle.go

```diff
@@ -197,6 +200,62 @@ func MergeBundles(a, b *common.Bundle) (*common.Bundle, bool) {
 	return c, changed
 }
 
+// PruneBundle removes the bundle RootCAs and JWT keys that expired before a given time
+// It returns an error if prunning results in a bundle with no CAs or keys
+func PruneBundle(bundle *common.Bundle, expiredBefore int64, log hclog.Logger) (*common.Bundle, error) {
```

Comment:
I think it is clearer to take `time.Time` instead of `int64` and let callers figure out how to convert whatever they have into a `time.Time` instead of leaving room for confusion on the time units `expiredBefore` represents.